README.md

```diff
@@ -20,14 +20,15 @@ The stack is composed of:
     - [Developing a new DAG](#developing-a-new-dag)
     - [Tags](#tags)
     - [Moving to production](#moving-to-production)
+      - [If there has been a change to the code in the repo:](#if-there-has-been-a-change-to-the-code-in-the-repo)
```

Comment:
Like, last night, since we were doing something that was purely a stack update via the image, we could have left the changes in the repo un-pulled down onto `01`, but I think we'd just be leaving ourselves a tiny foot gun there .. I can't think of a situation where we'd want to update the image but leave outdated `Dockerfile` or compose files in production.

I guess what I'm getting at -- I think a `git pull` is prudent both when you're doing a "normal" update where you have changed or added a dag, and also we should include a `git pull` when the stack is being updated, just so that it's up to date with HEAD on the origin. 

Maybe change this line on 23 to read "Steps for all updates, both to DAGs and to the stack" or something like that?

Reply:
ahh yes! that eliminates the weirdness that i was feeling about these two separate paths having some overlap that was only suggested. 🙏 i'll add `git pull` explicitly to the second section so it will always be in the steps that happen when the stack changes from maintenance or other needs.